azavea/pwd-waterworks-revealed
# Restructure workflow for new content

These changes bring the new content into workflow of the zones and remove the old content. Also, since the new workflow is simpler, lots of unneeded code was removed.

See the commit messages for details.

A few issues were created as a result of these changes: #49, #50, #51, #52.

To test:
- Load the app and enter some of the zones. You should see the new images.

Connects to #34
